src/app/public/modules/ag-grid/types/sky-component-properties.ts

```diff
@@ -0,0 +1,6 @@
+import { NumericOptions } from '@skyux/core';
+
+export interface SkyComponentProperties extends NumericOptions {
```

Comment:
I don't know that pulling this out into its own type can be done like this. This works for Currency and Numeric; however, Autocomplete and Datepicker also take in a param called `skyComponentProperties` but which use different properties than `NumericOptions`. 

So having a class called `SkyComponentProperties` which can't be used everywhere we have sky component properties passed in I believe will just cause confusion going forward.

Reply:
This is also somewhat changing the hierarchy that we had before

SkyCurrencyParms was extending the validator params to get validator information and had the component properties to get currency information

Now SkyCurrencyParams is extending the validator params which give the component information and then the component information give the validation information.  This means that the validator params really aren't needed as we could just move that one property up to the SkyCurrencyParams.

Reply:
I understand trying to clean up for a common API, but I am just concerned that we are cleaning it up somewhat for currency and numeric but that is coming at the cost of not being consistent with our other renders that take in component information. Happy to chat if you'd like.